### PR TITLE
simplify pom by using simple property rather than xml fragment

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <oliver.lamy@gmail.com>

### CODE REVIEW

File: jetty-http3/http3-tests/pom.xml [hunk 1]
```diff
@@ -10,6 +10,11 @@
   <artifactId>http3-tests</artifactId>
   <name>Jetty :: HTTP3 :: Tests</name>
 
+  <properties>
```
Comment (olamy) [inline jetty-http3/http3-tests/pom.xml:13]: @sbordet I assume we should not deploy those tests as part of the release and was a mistake? or I am wrong?

File: jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml [hunk 2]
```diff
@@ -70,14 +60,6 @@
           </archive>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-deploy-plugin</artifactId>
-        <configuration>
-          <!-- must not deploy -->
```
Comment (olamy) [inline jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml:77]: I have to say I was confused here and not sure to believe the comment or the code. :)

File: jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml [hunk 3]
```diff
@@ -70,14 +60,6 @@
...
-          <skip>false</skip>
-        </configuration>
```
Comment (olamy) [inline jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml:79]: I have to say I was confused here and not sure what to believe the comment or the code. :)

File: jetty-http3/http3-tests/pom.xml (hunk 1 above)
Comment (sbordet) [reply jetty-http3/http3-tests/pom.xml:13]: This module will only produce a test jar (no sources), so it won't be deployed anyway, right?
Ah, I see it deployed an empty jar 😒 
So yes let's skip deploy for this module.

File: jetty-websocket/websocket-jetty-tests/pom.xml [hunk 4]
```diff
@@ -12,6 +12,7 @@
 
   <properties>
     <bundle-symbolic-name>${project.groupId}.tests</bundle-symbolic-name>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
```
Comment (sbordet) [inline jetty-websocket/websocket-jetty-tests/pom.xml:16]: Should not the deploy be skipped too?

**Review by sbordet — CHANGES_REQUESTED** [review]

File: jetty-websocket/websocket-jetty-tests/pom.xml (hunk 4 above)
Comment (olamy) [reply jetty-websocket/websocket-jetty-tests/pom.xml:16]: correct I forgot that

Comment (olamy) [reply jetty-websocket/websocket-jetty-tests/pom.xml:16]: fixed https://github.com/eclipse/jetty.project/pull/7580/commits/10a9706d6ddd9c24767759742fbf2249d8d2b028

File: jetty-http3/http3-tests/pom.xml (hunk 1 above)
Comment (olamy) [reply jetty-http3/http3-tests/pom.xml:13]: after digging a bit the jar is not empty as it included some content we put in `target/classes` as part of the build such: generated MANIFEST.MF, LICENSE and NOTICE.txt

**Review by sbordet — DISMISSED** [review]

File: jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml (hunk 3 above)
Comment (janbartel) [reply jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml:79]: I think it should be the same as the other osgi test projects ;)

File: tests/jetty-jmh/pom.xml [hunk 5]
```diff
@@ -57,18 +57,6 @@
...
       </plugin>
     </plugins>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-deploy-plugin</artifactId>
-          <configuration>
-            <!-- No point deploying jmh project -->
-            <skip>true</skip>
```
Comment (janbartel) [inline tests/jetty-jmh/pom.xml:67]: Needs the property defined? Or is that covered by the toplevel tests/pom.xml properties?

**Review by janbartel — CHANGES_REQUESTED** [review]

File: tests/jetty-jmh/pom.xml (hunk 5 above)
Comment (olamy) [reply tests/jetty-jmh/pom.xml:67]: correct. properties are inherited.

**Review by sbordet — DISMISSED** [review]

**Comment by janbartel** [comment]
@olamy this one needs merge conflicts fixed

**Review by janbartel — CHANGES_REQUESTED** [review]
Changes themselves look ok, but this branch needs re-merging from jetty-10.0.x.

**Comment by janbartel** [comment]
@olamy nudge, needs merge.

**Comment by olamy** [comment]
@janbartel oh yes I definitely forgot this old PR. thanks for the ping :)

**Review by janbartel — APPROVED** [review]

**Pull request merged**